examples/pubsub/server_pubsub_subscriber_rt_level.c

```diff
@@ -90,6 +103,42 @@ SubscribeAfterWriteCallback(
     UA_String_clear(&strId);
 }
 
+// callback gets triggered after subscriber has received data
+// received data hasn't been copied/handled yet
+static void 
+SubscriberWriteCallback(
+    UA_Server *ipServer,
```

Comment:
Why ipServer?
We don't use Hungarian Notation in the other examples.
Let's stay consistent.